src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/AdminMenu.cs

```diff
@@ -22,8 +22,7 @@ public Task BuildNavigationAsync(string name, NavigationBuilder builder)
             }
 
             builder
-                .Add(S["Configuration"], NavigationConstants.AdminMenuConfigurationPosition, design => design
-                    .AddClass("menu-configuration").Id("configuration")
```

Comment:
Why removing the constant?

Reply:
It's only needed in the settings module since that one is what define the position, the is for the view.

Reply:
Maybe the idea is that if the settings module is not enabled the configuration menu is still well positioned, anyway in most other modules this constant is not used when defining the Configuration menu item ;), hmm but make sense if these modules depends on OC.Settings.

Reply:
The settings module is always enabled too.